apps/koala-frontend/src/app/shared/dates.validator.ts

```diff
@@ -0,0 +1,14 @@
+import { AbstractControl, ValidationErrors, ValidatorFn } from '@angular/forms';
+
+export const datesStartEndValidator: ValidatorFn = (control: AbstractControl): ValidationErrors | null => {
+  const startDate: Date = control.get('start')?.value;
+  const endDate: Date = control.get('end')?.value;
+
+  if (startDate && endDate) {
+    return startDate <= endDate ? null : { datesWrong: true };
```

Comment:
wouldn't it make more sense to return here the actual error message, so that it can be displayed in the ui?!

Reply:
Problem is that we currently do not have custom message possibility on input fields. Also in the angular tutorial, they use an indicator and leave the output message to the view.